pysteps/tests/test_plt_cartopy.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from pysteps.visualization import plot_precip_field
+from pysteps.utils import to_rainrate
+from pysteps.tests.helpers import get_precipitation_fields
+import matplotlib.pyplot as pl
+
+pytest.importorskip("cartopy")
+
+plt_arg_names = ("source", "map", "drawlonlatlines", "lw")
+
+plt_arg_values = [
+    ("mch", "cartopy", False, 0.5),
+    ("mch", "cartopy", True, 1.0),
+    ("bom", "cartopy", True, 0.5),
+    ("fmi", "cartopy", True, 0.5),
+    # TODO: why isn't the knmi projection workin? ("knmi", "cartopy", True, 0.5),
```

Comment:
@RubenImhoff, tests fail when plotting KNMI data with cartopy, can you maybe have a look into it ?

Reply:
I guess it has to do with the somewhat odd projection of the KNMI data. I'll try to have a look at it soon!

Reply:
I hope to have some time tomorrow! :)

Reply:
Hi @dnerini. We are here just testing if pysteps can plot. We are not assessing if what it was plotted is what was expected to be plotted, right?

Reply:
Hi @cvelascof, that's right, currently we test only if the plotting functions produce no errors, a sort of first basic test...

Reply:
Problem found, the problem is in the KNMI importer (or better said, the fact that the projection is expecting kilometers instead of meters, while the data is meant to be in meters). However, changing the meters to kilometers (so the pixelsize too) fixes it. I will do a little more testing and then I'll change it tomorrow. Maybe it's better if I do that in a different PR, as it changes some of the tests too etc.?

Reply:
Well done @RubenImhoff !
Yes, it probably makes more sense to open a different PR as you say.
I just wonder about changing the projection definition into km wil cause problems somewhere else, as we always assumed that the unit would be meters. need to think a bit more about that...

Reply:
I was thinking the same.. So far I haven't seen any differences, but I'm not sure enough to affirm that it should work. Option B would be to change the x- and y-values of the metadata in the plotting/plot-testing scripts (should work too).